specs/ContextMenuRequested.md

```diff
@@ -0,0 +1,764 @@
+        COREWEBVIEW2_CONTEXT_MENU_ITEM_DESCRIPTOR_CUT,
+
+        /// Context menu item descriptor for "Copy" action.
+        COREWEBVIEW2_CONTEXT_MENU_ITEM_DESCRIPTOR_COPY,
+
+        /// Context menu item descriptor for "Paste" action.
+        COREWEBVIEW2_CONTEXT_MENU_ITEM_DESCRIPTOR_PASTE,
+
+        /// Context menu item descriptor for "Paste as plain text" action.
+        COREWEBVIEW2_CONTEXT_MENU_ITEM_DESCRIPTOR_PASTE_AS_PLAIN_TEXT,
+
+        /// Context menu item descriptor for "Select all" action.
+        COREWEBVIEW2_CONTEXT_MENU_ITEM_DESCRIPTOR_SELECT_ALL,
+
+        /// Context menu item descriptor for "Open link in new window" action.
+        COREWEBVIEW2_CONTEXT_MENU_ITEM_DESCRIPTOR_OPEN_LINK_NEW_WINDOW,
+
+        /// Context menu item descriptor for "Save link as" action.
+        COREWEBVIEW2_CONTEXT_MENU_ITEM_DESCRIPTOR_SAVE_LINK_AS,
+
+        /// Context menu item descriptor for "Copy link" action.
+        COREWEBVIEW2_CONTEXT_MENU_ITEM_DESCRIPTOR_COPY_LINK,
+
+        /// Context menu item descriptor for "Save image as" action.
+        COREWEBVIEW2_CONTEXT_MENU_ITEM_DESCRIPTOR_SAVE_IMAGE_AS,
+
+        /// Context menu item descriptor for "Copy image" action.
+        COREWEBVIEW2_CONTEXT_MENU_ITEM_DESCRIPTOR_COPY_IMAGE,
+
+        /// Context menu item descriptor for "Copy image link" action.
+        COREWEBVIEW2_CONTEXT_MENU_ITEM_DESCRIPTOR_COPY_IMAGE_LINK,
+
+        /// Context menu item descriptor for "Save media as" action.
+        COREWEBVIEW2_CONTEXT_MENU_ITEM_DESCRIPTOR_SAVE_MEDIA_AS,
+
+        /// Context menu item descriptor for other commands not corresponding with any of the enums above.
+        COREWEBVIEW2_CONTEXT_MENU_ITEM_DESCRIPTOR_OTHER,
+    } COREWEBVIEW2_CONTEXT_MENU_ITEM_DESCRIPTOR;
+    
+    /// Indicates the kind of context for which the context menu was created
+    /// for the `ICoreWebView2ContextMenuInfo::get_ContextKind` method
+    [v1_enum]
+    typedef enum COREWEBVIEW2_CONTEXT_KIND
+    {
+        /// Indicates that the context menu was created for the page without any additional content.
+        COREWEBVIEW2_CONTEXT_KIND_PAGE,
+
+        /// Indicates that the context menu was created for an image element.
+        COREWEBVIEW2_CONTEXT_KIND_IMAGE,
+
+        /// Indicates that the context menu was created for selected text.
+        COREWEBVIEW2_CONTEXT_KIND_SELECTED_TEXT,
+        
+        /// Indicates that the context menu was created for an audio element.
+        COREWEBVIEW2_CONTEXT_KIND_AUDIO,
+        
+        /// Indicates that the context menu was created for a video element.
+        COREWEBVIEW2_CONTEXT_KIND_VIDEO,
+    } COREWEBVIEW2_CONTEXT_KIND;
+
+    /// Specifies the menu item kind
+    /// for the `ICoreWebView2StagingContextMenuItem::get_Kind` method
+    [v1_enum]
+    typedef enum COREWEBVIEW2_CONTEXT_MENU_ITEM_KIND
+    {
+        /// Specifies a normal menu item kind.
+        COREWEBVIEW2_CONTEXT_MENU_ITEM_KIND_NORMAL,
```

Comment:
I think "normal" is a bit judgmental. It suggests that the others are abnormal.

Perhaps call this one "Command".